src/library/scala/runtime/LazyRef.scala

```diff
@@ -10,7 +10,7 @@ package scala.runtime
 
 /** Classes used as holders for lazy vals defined in methods. */
 
-class LazyRef[T] {
+class LazyRef[T] extends Serializable {
```

Comment:
Let's add the `@SerialVersionUID` annotations on these in case we need to change the representation while having the ability to be serialization backwards compatible.

Reply:
ah yes, thanks